corehq/apps/hqwebapp/static/hqwebapp/js/select2_knockout_bindings_v3.ko.js

```diff
@@ -1,11 +1,18 @@
 hqDefine("hqwebapp/js/select2_knockout_bindings_v3.ko", [
-], function () {
+    'jquery',
+    'knockout',
+    'DOMPurify/dist/purify.min',
+], function (
+    $,
+    ko,
+    DOMPurify
+) {
     /**
      * Converts the bound element to a select2 widget. The value of the binding is
      * a list of strings, or a list of objects with the keys 'id' and 'text' used
      * for the select2's options.
      */
-    ko.bindingHandlers.select2 = new function () {
+    ko.bindingHandlers.select2 = function () {
         var that = this;
```

Comment:
In context of removing `new` from `new function ()` (which is hideous). I think it will work if you change this to `var that = {}` and make this function `return that;`

Reply:
yes, that's much better, thank you for thinking through that - cleaned up this code in 54e338e1ee0